python/surf/misc/_McsReader.py

```diff
@@ -86,19 +91,19 @@ def open(self, filename):
                     else:
                     
                         strings = [line[j:j+2] for j in range(1, len(line), 2)]
-                        bytes = [int(s, 16) for s in strings]
+                        hexBytes = [int(s, 16) for s in strings]
```

Comment:
I think you could skip having the strings with something like this:
```
hexBytes = [int(line[j:j+2], 16) for j in range(1, len(line), 2)]
```

And where you need strings, which is only to make the `baseAddr` you could just pick the indexes for it. 
Something like this:
```
baseAddr = int(line[10:11]+lines[12:13], 16)* (2**16)
```

This would make it one less iteration over `line` for every line at the file and the allocation of a list every time just so you can assemble the `baseAddr` later on...

Reply:
Applied your recommendation in 1d40657

Reply:
Applied your recommendation in d35c514